nifi-registry-client/src/main/java/org/apache/nifi/registry/client/impl/JerseyBucketClient.java

```diff
@@ -140,7 +150,8 @@ public Fields getFields() throws NiFiRegistryException, IOException {
                 target = target.queryParam("sort", sortParam.toString());
             }
 
-            return Arrays.asList(target.request().get(Bucket[].class));
+            return getRequestBuilder(target).get(List.class);
```

Comment:
I think this should be done as above in the other getAll() call no? Getting a Bucket[] and then returning emptyList or Arrays.asList?